tool/sync_default_gems.rb

```diff
@@ -371,7 +371,7 @@ def sync_default_gems(gem)
       `git checkout ext/digest/depend ext/digest/*/depend`
     when "set"
       sync_lib gem, upstream
-      cp_r(Dir.glob("#{upstream}/test/*"), "test/set")
+      cp_r("#{upstream}/test", ".")
```

Comment:
This revert previous changes,

Reply:
thanks :pray: https://github.com/ruby/ruby/pull/8228/commits/c7f3913ca772dfe990f45337cb87edcfb83b3b14